dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -33,9 +34,21 @@ const BLACK = neutral[7];
 const blogsGrayBackgroundPalette = (format: ArticleFormat): string => {
 	switch (format.theme) {
 		case ArticlePillar.News:
-			return pillarPalette[format.theme].main;
-		default:
-			return pillarPalette[format.theme].dark;
+			return news[400];
+		case ArticlePillar.Opinion:
+			return opinion[300];
+		case ArticlePillar.Sport:
+			return sport[300];
+		case ArticlePillar.Culture:
+			return culture[300];
+		case ArticlePillar.Lifestyle:
+			return lifestyle[300];
+		case ArticleSpecial.SpecialReport:
+			return specialReport[300];
+		case ArticleSpecial.SpecialReportAlt:
+			return news[400];
+		case ArticleSpecial.Labs:
+			return labs[300];
```

Comment:
This seems like a big change, have we verified these are the same colours?

Reply:
I guess there’s Chromatic for that… but it would be good to extract this change into its own PR.

Reply:
My answer to this question will attempt to answer also why we have so many `format.design !== ArticleDesign.Deadblog && format.design !== ArticleDesign.Liveblog`.

This PR introduces a new theme but I don't know for sure what designs editorial will decide to use. Possibly it will only be Standard, Comment, Analysis and Explainer. In (the unlikely) case though they decide to do a Liveblog / Deadblog, these looked particularly off and was agreed with @HarryFischer to make sure they're going to look like news. 

This code is an example. If I revert the change to what it was title and meta looked like this:

![image](https://user-images.githubusercontent.com/19683595/199588457-662b9b44-63b8-4664-860b-611f2c325410.png)

because:

```
switch (format.theme) {
		case ArticlePillar.News:
			return pillarPalette[format.theme].main;
		default:
			return pillarPalette[format.theme].dark;
	}
```

is equivalent to:

```
switch (format.theme) {
		case ArticlePillar.News:
			return news[400];
		case ArticlePillar.Opinion:
			return opinion[300];
		case ArticlePillar.Sport:
			return sport[300];
		case ArticlePillar.Culture:
			return culture[300];
		case ArticlePillar.Lifestyle:
			return lifestyle[300];
		case ArticleSpecial.SpecialReport:
			return specialReport[300];
		case ArticleSpecial.SpecialReportAlt:
			return specialReportAlt[100];
		case ArticleSpecial.Labs:
			return labs[300];
	}
```	

and `specialReportAlt[100]` is `#2B2B2A`, the gray in the screenshot. What we really wanted though was `news[400]`. For the same reason many colours were completely off, e.g. orange and red backgrounds in liveblog etc.

However, I realised that if end up covering all the cases in switch statements for all the components that were off that would be a very bog refactoring in many places as you say. So I decided for now to add

```
if (
		format.theme === ArticleSpecial.SpecialReportAlt &&
		format.design !== ArticleDesign.LiveBlog &&
		format.design !== ArticleDesign.DeadBlog
	)
```

so that at least:
1. We get the result we want for `SpecialReportAlt`
2. We don't risk breaking other themes / designs due to large refactoring in many places.

The downside is that this is indeed fiddly but I am planning to revisit `decidePalette` and do these refactorings in chunks that will allow us to gradually get rid of the deprecated `pillarPalette` and rationalise the logic in the methods that have gone indeed wild.

Sorry, that's a lot of text! Thanks for asking these questions, I'm glad I got the opportunity to communicate what was only in my head and realising I should have written this in this PR in the first place! I hope this makes sense and very happy to chat about this as well - let me know what you think!

Reply:
Here's how SpecialReportAlt Liveblog used to look like

![image](https://user-images.githubusercontent.com/19683595/199592449-e0c0cd0c-ea3c-4308-9094-5b44d1a550bc.png)

Reply:
I guess this also demonstrates it is not good to code without specific use cases but that's a different story!